PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -101,16 +163,134 @@ def miniAOD_customizeCommon(process):
     process.patJetsAK8.tagInfoSources = cms.VInputTag(cms.InputTag("caTopTagInfosPAT"))
     process.patJetsAK8.addTagInfos = cms.bool(True)
 
-
-
-    # add Njetiness
+    # Add Njettiness
     process.load('RecoJets.JetProducers.nJettinessAdder_cfi')
     process.NjettinessAK8 = process.Njettiness.clone()
     process.NjettinessAK8.src = cms.InputTag("ak8PFJetsCHS")
     process.NjettinessAK8.cone = cms.double(0.8)
     process.patJetsAK8.userData.userFloats.src += ['NjettinessAK8:tau1','NjettinessAK8:tau2','NjettinessAK8:tau3']
 
+    ## Add soft drop subjet info
+    process.ak8PFJetsCHSSDSubJet = cms.EDProducer("RecoJetDeltaRValueMapProducer",
+        src = cms.InputTag("ak8PFJetsCHS"),
+        matched = cms.InputTag("selectedPatJetsAK8PFCHSSoftDropPacked"),
+        distMax = cms.double(0.8),
+        values = cms.vstring(
+            "? numberOfDaughters > 0 ? daughterPtr(0).pt : -9999",
```

Comment:
I'm not sure 9999 is a good pattern for gzip compression, probably a zero would produce a better compression. Is there really the need to store a p4 by value? why not storing a PackedCandidate collection with the daughters and rekey it?